lib/repositories/providers/agenda_provider.dart

```diff
@@ -12,7 +11,30 @@ class AgendaProvider {
     if (response.statusCode == 200) {
       return (response.data as List).map((json) => Event.fromJson(json)).toList();
     } else {
-      throw Exception('Impossible de récupérer les clubs');
+      throw Exception('Impossible de récupérer les événements');
+    }
+  }
+
+  Future<List<Event>> listTeamMonthMatches(String teamCode) async {
+    Response response = await dio.get("/equipes/$teamCode/matchs", options: buildConfigurableCacheOptions());
```

Comment:
Is there a way in the API to set the start and end date of the match we want to retrieve? Here we will retrieve **all** matches and then we filter the result, which is inefficient for the network. 
If it does not exist, can you add an issue for @Lo0ping and add a link to this line to fix it when the API will accept a start and end date.

Reply:
I'm quite sure it does not exist. I'll add an issue.

Reply:
Not it exists 😁

Reply:
@Lo0ping Can you provide more information, examples on how to query matches for a team with a start and an end date please?